CHANGELOG.md

```diff
@@ -97,11 +99,59 @@ The following libraries are now considered stable and complete:
 * [Secret Manager API](google/cloud/secretmanager/README.md) [[quickstart]](google/cloud/secretmanager/README.md)
 * [Cloud Tasks API](google/cloud/tasks/README.md) [[quickstart]](google/cloud/tasks/README.md)
 
+## [BigQuery](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/README.md)
+
+* feat(generator): add options support to generated clients ([#7683](https://github.com/googleapis/google-cloud-cpp/pull/7683))
+
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+* feat(pubsublite): stream factories for PSL bidi streams ([#7671](https://github.com/googleapis/google-cloud-cpp/pull/7671))
```

Comment:
Please remove the pubsublite change, it was just some test refactoring in Bigtable, and a bit confusing to add this in the Bigtable list.